packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -0,0 +1,195 @@
+export type ChipStyleType = (typeof CHIP_TYPES)[keyof typeof CHIP_TYPES];
+
+export type SingleSelectChipItem = {
+  text: string;
+  accessibilityLabel?: string;
+  leadingAccessoryView?: ReactNode;
+  className?: string;
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
+};
+
+export type ChipItem = {
+  component?: (props: BpkSelectableChipProps) => ReactElement;
+  onClick?: (selected: boolean, index: number) => void;
+  selected?: boolean;
+} & SingleSelectChipItem;
+
+export type CommonProps = {
+  ariaLabel?: string;
+  ariaLabelledBy?: string;
+  type: ChipGroupType;
+  className?: string | null;
+  style?: ChipStyleType;
+};
+
+export type ChipGroupProps = {
+  chips: ChipItem[];
+  stickyChip?: ChipItem;
+  _isSingleSelect?: boolean; // TODO: is there a better way for internal only props?
+} & CommonProps;
+
+const BpkChipGroup = ({ _isSingleSelect = false, ariaLabel, ariaLabelledBy, chips, className = null, stickyChip, style = CHIP_TYPES.default, type = CHIP_GROUP_TYPES.rail }: ChipGroupProps) => {
+  const scrollContainerRef = useRef<HTMLElement | null>(null);
+
+  // TODO: is there a way in typescript to enforce this instead?
+  if ((!ariaLabel && !ariaLabelledBy) || (ariaLabel && ariaLabelledBy)) {
+    console.warn('BpkChipGroup: Exactly one of ariaLabel and ariaLabelledBy should be set')
+  }
+
+  const containerClassnames = getClassName(
+    className,
+    'bpk-chip-group-container',
+  )
+
+  const chipGroupClassNames = getClassName(
+    'bpk-chip-group',
+    `bpk-chip-group--${type}`,
+  );
+
+  const stickyChipContainerClassnames = getClassName(
+    'bpk-sticky-chip-container',
+    `bpk-sticky-chip-container--${style}`,
+  );
+
+  const stickyChipClassnames = getClassName(
+    'bpk-sticky-chip',
+    stickyChip && stickyChip.className,
+  );
+
+  const scrollContainerLeftIndicator = getClassName(
+    'bpk-chip-group-scroller-left-indicator',
+    `bpk-chip-group-scroller-left-indicator--${style}`,
+  );
+
+  const scrollContainerRightIndicator = getClassName(
+    'bpk-chip-group-scroller-right-indicator',
+    `bpk-chip-group-scroller-right-indicator--${style}`,
+  );
+
+  const renderChipItem = ({ accessibilityLabel, component: Component = BpkSelectableChip, onClick, selected, text, ...rest }: ChipItem, index: number) => (
+    <Component
+      key={text}
+      selected={selected ?? false}
+      type={style}
+      accessibilityLabel={accessibilityLabel || text}
+      onClick={() => {
+        if (onClick) {
+          onClick(!selected, index);
+        }
+      }}
+      role="option"
+      {...rest}
+    >
+      {text}
+    </Component>
+  );
+
+  // TODO: Fix/remove the scroll indicator when style=CHIP_TYPES.onImage
+  // TODO: Fix shadows + focus indicators being cutoff when type = rail
+  const wrapRailInScroll = (children: ReactElement) =>
+    type === CHIP_GROUP_TYPES.rail ? (
+      <BpkMobileScrollContainer
```

Comment:
This BpkMobileScrollContainer seems like a nice way of making horizontal scrolls but it also causes a couple of issues in this case:

1. It adds this scrolling indicator which is nice as long as you've got the right background colour. So far I have kept it white for default chip style, and matched the on dark color to our contrast background color. However this would really restrict the backgrounds this can be used on. I'm thinking the simple solution would be to have a prop to disable these? 
![image](https://github.com/Skyscanner/backpack/assets/22656572/e4af9567-4dba-409e-a2bc-d2478b5272ec)


2. The focus indicator gets cutoff. I've spent some time trying to get this working as expected but without much luck. We'll need to spend more time to solve this. We see the same problem affecting all of the `onImage` chips as they use a shadow border.
<img width="304" alt="image" src="https://github.com/Skyscanner/backpack/assets/22656572/432ae78c-80ec-4288-87d0-87c3da35974e">
<img width="304" alt="image" src="https://github.com/Skyscanner/backpack/assets/22656572/7031bad5-969d-4b75-9972-38a4168f1309">

Reply:
These issues will be addressed by [DTO-5125](https://skyscanner.atlassian.net/browse/DTO-5125). From design: the total height should be 36px giving a 2px padding on the top and bottom which will make the shadows and hopefully this indicator visible